presto-main/src/main/java/com/facebook/presto/memory/MemoryResource.java

```diff
@@ -48,4 +50,12 @@ public MemoryInfo getMemoryInfo(MemoryPoolAssignmentsRequest request)
         taskManager.updateMemoryPoolAssignments(request);
         return memoryManager.getInfo();
     }
+
+    @GET
+    @Produces(MediaType.APPLICATION_JSON)
+    public MemoryInfo getMemoryInfo()
+    {
+        taskManager.updateMemoryPoolAssignments(new MemoryPoolAssignmentsRequest("", 0, new ArrayList<>()));
```

Comment:
Sure. Just made change to take out call to updateMemoryPoolAssignment() so the new implementation looks like this

```
@GET
    @Produces(MediaType.APPLICATION_JSON)
    public MemoryInfo getMemoryInfo()
    {
        taskManager.updateMemoryPoolAssignments(new MemoryPoolAssignmentsRequest("", 0, new ArrayList<>()));
        return memoryManager.getInfo();
    }
```
Also added TestMemoryResource.java with only one method that is 

```
@Test(singleThreaded = true)
public class TestMemoryResource
{
    private TestingPrestoServer server;
    private HttpClient client;

    @BeforeMethod
    public void setup()
            throws Exception
    {
        server = new TestingPrestoServer();
        client = new JettyHttpClient();
    }

    @AfterMethod
    public void teardown()
    {
        closeQuietly(server);
        closeQuietly(client);
    }

    @Test
    public void testGetMemoryInfo()
            throws Exception
    {
        MemoryInfo memoryInfo = client.execute(
                prepareGet().setUri(server.resolve("/v1/memory")).build(),
                createJsonResponseHandler(jsonCodec(MemoryInfo.class)));

        assertNotNull(memoryInfo);
        assertNotNull(memoryInfo.getTotalNodeMemory());
        assertNotNull(memoryInfo.getPools());
    }
}
```

I can also write a test case for POST method, under a different pull request. Can someone help me by creating an issue?